xbmc/playlists/PlayListXSPF.cpp

```diff
@@ -0,0 +1,150 @@
+/*
+ *  Copyright (C) 2018 Tyler Szabo
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "PlayListXSPF.h"
+
+#include "utils/log.h"
+#include "utils/URIUtils.h"
+#include "utils/XBMCTinyXML.h"
+#include "URL.h"
+
+using namespace PLAYLIST;
+
+/*
+
+Specification: http://www.xspf.org/specs/
```

Comment:
Do we want this in our codedocs? If so, then it should be in doxygen format in the header.

Reply:
I have a mixed opinion on it. I don't know the header is the right place because it doesn't provide much information to the caller (the caller doesn't need to know how to parse this) it's helpful to future editors to know what the document is expected to look like (see the other Playlist cpp files). However, because there's the test case we have an example (that, IMO is in a better location). I'll remove it but it might be worth adding back in for consistency among other playlists.